### PR TITLE
F/ Donation + Header Mobile & Fixes Grammar and semantic issues

### PR DESCRIPTION
**Pull request by mateodaza**
Merge first #201 and #207

### CODE REVIEW

File: src/utils/index.tsx [hunk 1]
```diff
@@ -29,6 +31,41 @@ export function pollEvery(fn: Function, delay: any) {
...
+		// check width if device not found
+		if (!mobile && typeof window !== 'undefined') {
+			mobile = window.innerWidth <= deviceSize.mobileL;
+		}
+		setMobile(mobile);
+	}, []);
+
+	return { isMobile };
+}
+
```
Comment (MohammadPCh) [inline src/utils/index.tsx:68]: Please move these hooks to the hooks folder.

File: src/components/views/donate/TokenPicker.tsx [hunk 2]
```diff
@@ -190,85 +212,119 @@ const TokenPicker = (props: {
...
+					backgroundColor: 'white',
+					borderRadius: 4,
+					boxShadow: `0 0 0 1px ${shadow}, 0 4px 11px ${shadow}`,
+					marginTop: 8,
+					position: 'absolute',
+					zIndex: 2,
+				}}
+				{...props}
+			/>
+		</>
```
Comment (MohammadPCh) [inline src/components/views/donate/TokenPicker.tsx:327]: Why do you add it to the fragment?

File: src/components/views/donate/DonateIndex.tsx [hunk 3]
```diff
@@ -159,32 +166,60 @@ const ProjectsIndex = (props: IProjectBySlug) => {
...
+									<H4>Donate With</H4>
+									<TypeSelection />
+								</>
+							)}
+						</Right>
+					</Sections>
+					{!isSuccess && !isMobile && <SocialBox project={project} />}
+				</Wrapper>
+			</Container>
+		</>
```
Comment (MohammadPCh) [inline src/components/views/donate/DonateIndex.tsx:222]: Why do you add fragment?

File: src/components/modals/InsufficientFund.tsx [hunk 4]
```diff
@@ -42,6 +43,21 @@ const InsufficientFundContainer = styled.div`
...
+	}
+	${mediaQueries['mobileL']} {
+		width: 100%;
+	}
+	${mediaQueries['desktop']} {
+		width: 528px;
+	}
+	${mediaQueries['tablet']} {
+		width: 528px;
+	}
```
Comment (MohammadPCh) [inline src/components/modals/InsufficientFund.tsx:60]: you can remove some of them and write like this:
```
	${mediaQueries['desktop']} {
		width: 528px;
	}
```

File: src/components/modals/DonateModal.tsx [hunk 5]
```diff
@@ -380,6 +381,15 @@ const DonateContainer = styled.div`
...
 	margin: -30px 0 0 0;
+	${mediaQueries['mobileS']} {
+		width: 100%;
+	}
+	${mediaQueries['mobileM']} {
+		width: 100%;
+	}
+	${mediaQueries['mobileL']} {
+		width: 100%;
+	}
```
Comment (MohammadPCh) [inline src/components/modals/DonateModal.tsx:392]: div width is 100% as a default.

File: src/components/modals/ChangeNetwork.tsx [hunk 6]
```diff
@@ -45,10 +47,20 @@ const ChangeNetworkModalContainer = styled.div`
...
 	color: ${brandColors.giv[700]};
+	${mediaQueries['mobileS']} {
+		width: 100%;
+	}
+	${mediaQueries['mobileM']} {
+		width: 100%;
+	}
+	${mediaQueries['mobileL']} {
+		width: 100%;
+	}
```
Comment (MohammadPCh) [inline src/components/modals/ChangeNetwork.tsx:58]: write like this:
```
width: 100%;
${mediaQueries['tablet']} {
		width: 500px;
	}
```

File: src/components/modals/ChangeNetwork.tsx [hunk 7]
```diff
@@ -45,10 +47,20 @@ const ChangeNetworkModalContainer = styled.div`
...
 `;
 
 const Title = styled(H4)`
 	margin: 18px 0 24px;
+	color: ${brandColors.giv[700]};
```
Comment (MohammadPCh) [inline src/components/modals/ChangeNetwork.tsx:63]: Do you consider themes?

**Review by MohammadPCh — APPROVED** [review]
Thank you for your great job @mateodaza 
I add some change requests that can fix easily.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
Thank you for your great job @mateodaza 
I add some change requests that can fix easily.

**Comment by mateodaza** [comment]
I believe merge got some grammar issues back to homepage, I'll check again before merging

**Review by RamRamez — APPROVED** [review]

**Pull request merged**